src/test/java/com/wikia/webdriver/testcases/discussions/ZeroErrorStateTests.java

```diff
@@ -0,0 +1,230 @@
+package com.wikia.webdriver.testcases.discussions;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.ErrorMessages;
+import com.wikia.webdriver.elements.mercury.pages.discussions.PostDetailsPage;
+import com.wikia.webdriver.elements.mercury.pages.discussions.PostsListPage;
+import com.wikia.webdriver.elements.mercury.pages.discussions.ReportedPostsAndRepliesPage;
+import com.wikia.webdriver.elements.mercury.pages.discussions.UserPostsPage;
+
+import org.testng.annotations.Test;
+
+@Execute(onWikia = MercuryWikis.DISCUSSIONS_MESSAGING)
+@Test(groups = {"discussions-zero-error-state"})
+
+public class ZeroErrorStateTests extends NewTestTemplate {
+
+  private static final String DESKTOP_RESOLUTION = "1920x1080";
+
+  /**
+   * ANONS ON DESKTOP SECTION
+   */
+
+  @Test(groups = "discussions-anonUserOnDesktopSeeProperMessageWhenUploadsEmptyReportedPostsPage")
+  @Execute(asUser = User.ANONYMOUS)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void anonUserOnDesktopSeeProperMessageWhenUploadsEmptyReportedPostsPage() {
```

Comment:
pease update the verb to proper grammar form:
anonUser ... Sees
It's the third person so add "s" to the verb.
Here and in all other tests and in the method userSee (...) -> userSees (...)